updating/understanding_updates/intro-to-updates.adoc

```diff
@@ -0,0 +1,43 @@
+:_content-type: ASSEMBLY
+[id="understanding-openshift-updates"]
+= Introduction to OpenShift updates
+include::_attributes/common-attributes.adoc[]
+:context: understanding-openshift-updates
+
+toc::[]
+
+With {product-title} 4, you can update an {product-title} cluster with a single operation by using the web console or the OpenShift CLI (`oc`). Platform administrators are automatically notified when an update is available for their cluster.
```

Comment:
worth adding a word on how exactly they are notified?

Reply:
I can add that to this sentence or paragraph, assuming there aren't a ton of different ways. Could you provide a quick description of how the notifications work, like "through console alerts or notifications"?

Reply:
depending on what do we prefer to teach an admin to monitor.. `oc adm upgrade` output? something else? @petr-muller @wking wdyt?  

"automatically notified" sounds as if they get a knock on the door for that one ;)

Reply:
Are they actually notified though?  OSD sends you an email so for that case it is true, but I would not consider "new lines show up in web console or when you happen to run the right CLI command" an automatic notification.

Personally I would soften this to something like "new update options are offered to the administration in the `Administration -> Cluster Settings`  view in web console or in `oc adm upgrade` output"

Reply:
yeah I also inclined into softening it, so we won't end up committed to some "automatic notification" feature

Reply:
I will modify this to roughly copy what Petr suggested in his comment.